client/layout/color-scheme/index.jsx

```diff
@@ -10,11 +9,10 @@ export function getColorScheme( { state, isGlobalSidebarVisible, sectionName } )
 	if ( sectionName === 'checkout' ) {
 		return null;
 	}
-	const calypsoColorScheme = getPreference( state, 'colorScheme' );
 	const siteId = getSelectedSiteId( state );
 	const siteColorScheme = getAdminColor( state, siteId );
 
-	return siteColorScheme ?? calypsoColorScheme;
+	return siteColorScheme ?? 'default';
```

Comment:
No good reason to use the preference now since the site color schemes will prevail. Using the preference was actually causing unexpected behavior since folks wouldn't know where to change it.